### PR TITLE
pass is_auto param to worker builds

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
It would be really great to verify that the return value if `is_rebuild` is the value actually used.

**Comment by rcerven** [comment]
unit test added

File: tests/plugins/test_orchestrate_build.py [hunk 1]
```diff
@@ -823,3 +831,72 @@ def test_orchestrate_build_failed_to_list_builds(tmpdir, fail_at):
...
+
+    runner = BuildStepPluginsRunner(
+        workflow.builder.tasker,
+        workflow,
+        [{
+            'name': OrchestrateBuildPlugin.key,
+            'args': plugin_args,
+        }]
+    )
+    workflow.prebuild_results[CheckAndSetRebuildPlugin.key] = is_auto
```
Comment (twaugh) [inline tests/plugins/test_orchestrate_build.py]: This is the only time `is_auto` is used in this function. Is that intentional?

File: tests/plugins/test_orchestrate_build.py [hunk 2]
```diff
@@ -823,3 +831,72 @@ def test_orchestrate_build_failed_to_list_builds(tmpdir, fail_at):
...
+
+    for record in caplog.records():
+        if not record.name.startswith("atomic_reactor"):
+            continue
+
+        assert hasattr(record, 'arch')
+        if record.funcName == 'watch_logs':
+            assert record.arch == 'x86_64'
+        else:
+            assert record.arch == '-'
```
Comment (twaugh) [inline tests/plugins/test_orchestrate_build.py]: Most of this seems to be a duplicate of tests elsewhere.

File: tests/plugins/test_orchestrate_build.py (hunk 1 above)
Comment (rcerven) [inline tests/plugins/test_orchestrate_build.py]: it is used in expected_kwargs as well

Comment (twaugh) [inline tests/plugins/test_orchestrate_build.py]: Ok.

File: tests/plugins/test_orchestrate_build.py (hunk 2 above)
Comment (twaugh) [inline tests/plugins/test_orchestrate_build.py]: Can we remove the duplicate testing here? Looks like everything from line 873.

Comment (rcerven) [inline tests/plugins/test_orchestrate_build.py]: ok removed the same things, which were in different test and kept just the kwargs check

File: tests/plugins/test_orchestrate_build.py (hunk 1 above)
Comment (rcerven) [inline tests/plugins/test_orchestrate_build.py]: and the test is testing if create_worker receives expected arguments

**Comment by lcarva** [comment]
Updated release notes for 1.6.26.

**Pull request merged**